lib/diagnostic.js

```diff
@@ -6,9 +6,12 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
-};
+const normalizeWords = function (paragraph) {
+  let words = paragraph.split('');
```

Comment:
Splitting a string with an empty string will split it into an array of every character that the string consists of.